db/migrate/20170903200617_create_person.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePerson < ActiveRecord::Migration[5.1]
+  def change
+    create_table :people do |t|
+
+      t.string :name
+      t.date :bith
```

Comment:
I believe here the correct is `birthday`

Reply:
ok..I gonna change